api/app/Models/PoolCandidate.php

```diff
@@ -238,27 +238,22 @@ public function filterByLocationPreferences(Builder $query, ?array $workRegions)
         });
         return $query;
     }
+
```

Comment:
The languageAbility field still exists on PoolCandidate in the schema (even if deprecated) so PoolCandidate.php should also have an accessor for it. It can reference the User accessor.

Reply:
I didn't deprecate it on `PoolCandidate` though and didn't drop it from the pool candidate table
reason being the field is being taken care of in #5055 

![image](https://user-images.githubusercontent.com/40485260/207413200-3d709fca-2f8d-4ddb-a346-5814ac1a004f.png)

Reply:
Ah, that's right, you didn't actually drop it from the pool candidate table. Carry on. 😁